src/auslib/web/admin/swagger/api_v2.yml

```diff
@@ -0,0 +1,178 @@
+openapi: 3.0.0
+info:
+  title: Balrog Admin API
+  description: Balrog Admin API
+  version: 2.0.0
+
+paths:
+  /releases:
+    get:
+      summary: Get a list of basic information about Releases in Balrog's database
+      operationId: auslib.web.admin.views.releases_v2.get_releases
+      responses:
+        '200':
+          description: List of Releases
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - releases
+                properties:
+                  releases:
+                    type: array
+                    items:
+                      type: object
+                      required:
+                        - name
+                        - rule_info
+                      properties:
+                        name:
+                          type: string
+                        product:
+                          type: string
+                          nullable: true
+                        data_version:
+                          type: number
+                          nullable: true
+                        read_only:
+                          type: boolean
+                          nullable: true
+                        rule_info:
```

Comment:
This is largely because I want to see what the full spec looks like before trying to refactor it. In the old spec, we ended up with a lot of very similar but different entries for the same thing[1]. The next PR will complete the API, and I'll do a refactoring pass on the schema.

[1] Eg: product has 5 definitions: https://github.com/mozilla-releng/balrog/blob/master/src/auslib/web/common/swagger/definitions.yml#L87, https://github.com/mozilla-releng/balrog/blob/master/src/auslib/web/common/swagger/parameters.yml#L2, https://github.com/mozilla-releng/balrog/blob/master/src/auslib/web/common/swagger/parameters.yml#L11, https://github.com/mozilla-releng/balrog/blob/master/src/auslib/web/common/swagger/definitions.yml#L237, https://github.com/mozilla-releng/balrog/blob/master/src/auslib/web/admin/swagger/api.yml#L131

Reply:
Makes sense, thanks.